Directory.Build.props

```diff
@@ -27,4 +27,9 @@
     <BaseIntermediateOutputPath>$(IntermediateOutputPath)</BaseIntermediateOutputPath>
   </PropertyGroup>
 
+  <!-- Workaround nuget restore warnings turning into errors in the BuildTools tool-runtime/project.csproj -->
```

Comment:
Why can’t BuildTools do this?  This feels wrong

Reply:
After looking up `Directory.Build.props` (had no idea this was auto-imported) it sounds like we can create the file in the `tool-runtime` dir with this property and we will be ok. It actually seems like we *should* do this to prevent the repo from modifying the BuildTools init behavior. (And prevent the environment from playing a part, if the repo has no `Directory.Build.props`.)

Reply:
IMO - the root cause here is the warnings, which should be fixed.  We need to get BuildTools to a "clean" state, which it isn't in right now.  

These package mis-alignments have caused problems up and down repos and we keep pushing fixing them off.  This is just another case of them causing problems.

Reply:
I don't plan on checking in this workaround I want to make the change in buildtools, minimally to disable the warnings for this project but ideally fixing the packages. I just submitted this as a temp workaround before the EOD yesterday to try and flush out other issues in this merge.